pom.xml

```diff
@@ -48,14 +48,14 @@
 
         <version.org.slf4j>1.7.25</version.org.slf4j>
         <version.com.fasterxml.jackson>2.10.3</version.com.fasterxml.jackson>
-        <version.javax.validation>2.0.1.Final</version.javax.validation>
+        <version.jakarta.validation>3.0.2</version.jakarta.validation>
         <version.org.junit.minor>6.0</version.org.junit.minor>
         <version.org.junit>5.${version.org.junit.minor}</version.org.junit>
         <version.org.junit.jupiter>${version.org.junit}</version.org.junit.jupiter>
         <version.org.mockito>3.0.0</version.org.mockito>
         <version.ch.qos.logback>1.1.3</version.ch.qos.logback>
         <version.org.assertj>3.13.2</version.org.assertj>
-        <version.jsonschema2pojo-maven-plugin>1.0.1</version.jsonschema2pojo-maven-plugin>
+        <version.jsonschema2pojo-maven-plugin>1.1.2 </version.jsonschema2pojo-maven-plugin>
```

Comment:
can this please be done as a separate pr? dont think its tied to the jakarta stuff? thanks